cluster/proto/api/rbac_requests.go

```diff
@@ -15,8 +15,21 @@ import (
 	"github.com/weaviate/weaviate/usecases/auth/authorization"
 )
 
+const (
+	// in case changes happens to the RBAC message, add new version before the RBACLatestCommandPolicyVersion
+	// RBACCommandPolicyVersionV0 represents the first version of RBAC commands
+	RBACCommandPolicyVersionV0 = iota
```

Comment:
should we not start with 1?

because replayes of old `cmd.CreateRolesRequest` will not contain the version parameter and go will use the default value of 0. But for new comments we want to set it to 1 to be able to distinguish between new and old?